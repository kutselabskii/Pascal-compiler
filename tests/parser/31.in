program abc;

var a, i: integer;

begin
if (a > 0) then
	write(1);
else
	write(2);
end.