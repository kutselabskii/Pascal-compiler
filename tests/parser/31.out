Line 9 : Col 5 : Parser error :: 'end' expected
